SnapMD.ConnectedCare.ApiModels/PatientOnBoardShortDetail.cs

```diff
@@ -43,6 +43,7 @@ public class PatientOnBoardShortDetail
         public string Gender { get; set; }
         public PatientOnBoardStatus? Status { get; set; }
         public bool? PreventSendingInvitation { get; set; }
+        public bool? ResendRegistrationEmail { get; set; }
```

Comment:
@jay-lee00 This should be a separate API. If we already have the patient data in the DB we should not issue a "POST" call with that data because the request should be invalid.